gradle/publish.gradle

```diff
@@ -144,6 +146,8 @@ configure(scalaProjects) { project ->
 
     afterEvaluate {
         jar configureJarManifestAttributes(project)
+        publishing.publications.mavenJava.artifactId = project.archivesBaseName
```

Comment:
Does this result in a change to the POM? Or is it for consistency with Java section?

Reply:
For consistency with the Java section